pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -95,6 +110,30 @@ func (p *Pilot) syncFunc(pilot *v1alpha1.Pilot) error {
 	return nil
 }
 
+func run(args ...string) error {
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	os.Unsetenv("JVM_OPTS")
+	return cmd.Run()
+}
+
+func (p *Pilot) decommission() error {
+	nodes, err := p.nodeTool.Status()
+	if err != nil {
+		return err
+	}
+	localNode := nodes.LocalNode()
+
+	// if node is operational and working normally, decommission node
+	if localNode.State == nodetool.NodeStateNormal {
+		glog.Info("about to decomission node")
+		return run("nodetool", "decommission")
+	}
+
+	return nil
```

Comment:
I think we should return an error if the node state != NodeStateNormal.
Otherwise the unhealthy node will be marked as decommissioned in the pilot status, right?